rasqc/checkers/breaklines.py

```diff
@@ -0,0 +1,91 @@
+"""Checks related to breaklines within a HEC-RAS model."""
+
+from ..base_checker import RasqcChecker
+from ..registry import register_check
+from ..rasmodel import RasModel
+from ..result import RasqcResult, ResultStatus
+
+from rashdf import RasGeomHdf
+from pathlib import Path
+
+ENFORCEMENT_TOLERANCE = 5
+MIN_FLAG_LENGTH = 10
+
+
+@register_check(["ble"], dependencies=["GeomHdfExists"])
+class BreaklineEnforcement(RasqcChecker):
+    """Checker for breakline enforcement.
+
+    Checks the breakline enforcement within the current geometry
+    and returns a `GeoDataFrame` of delinquent breaklines.
```

Comment:
For checks like this that use constants like `ENFORCEMENT TOLERANCE` can we include a little more info about how the checks work? e.g., I think this one buffers the mesh faces, overlays them on top of the breaklines, then identifies any dangling breakline segments?